backend/migrations/20230703131255_food_plan.up.sql

```diff
@@ -0,0 +1,14 @@
+-- Add up migration script here
+
+CREATE DOMAIN price AS integer CHECK (VALUE >= 0);
+
+CREATE TABLE food_plan (
+  line_id uuid NOT NULL,
+  food_id uuid NOT NULL,
+  date date NOT NULL,
```

Comment:
`date date` => no compile error?

Reply:
it complied... but we can rename it to `serve_date`

Reply:
Das sind Sprachenabhängige Implementierungsdetails